packages/node/src/Session.ts

```diff
@@ -85,12 +86,39 @@ export const defaultStorage = new InMemoryStorage();
 /**
  * A {@link Session} object represents a user's session on an application. The session holds state, as it stores information enabling acces to private resources after login for instance.
  */
-export class Session extends EventEmitter {
+export class Session
+  extends SessionEventEmitter
+  implements ISessionEventEmitter
+{
   /**
    * Information regarding the current session.
    */
   public readonly info: ISessionInfo;
 
+  /**
+   * Session attribute exposing the EventEmitter interface, to listen on session
+   * events such as login, logout, etc.
+   */
+  public readonly events: SessionEventEmitter;
+
+  /**
+   * Proxy handler to ease transition to the Session no longer being an EvenEmitter,
+   * and instead Session["events"] being a SessionEventEmitter. For the time being,
+   * Session["events"] is just a proxy to the SessionEventEmitter interface of
+   * the Session.
+   *
+   * When Session no longer implements SessionEventEmitter, this can be removed.
+   * @hidden
+   */
+  private handler = {
+    // This proxy is only a temporary measure until Session no longer extends
+    // SessionEventEmitter, and the proxying is no longer necessary.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    get(target: any, prop: any, receiver: any) {
+      return Reflect.get(target, prop, receiver);
+    },
```

Comment:
My comments about erroring on non `EventEmitter` properties also applies to the node API.